python/py-hpack/Portfile

```
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        python-hyper hpack 3.0.0 v
+github.setup        python-hyper hpack 4.0.0 v
 name                py-hpack
 revision            1
```

The revision should drop to 0 when the version is increased.
```suggestion
revision            0
```